package.json

```diff
@@ -22,7 +22,9 @@
     "lint:sol-tests": "solhint -c .solhint.tests.json 'test/**/*.sol'",
     "prepare": "husky install",
     "test": "forge test -vvv",
+    "test:fuzz": "echidna test/invariants/fuzz/Greeter.t.sol --contract GreeterInvariant --corpus-dir test/invariants/fuzz/echidna_coverage/ --test-mode assertion",
```

Comment:
Is it possible to use wildcards to run against all contracts under `fuzz` ?

Reply:
check the previous comment from gas @0xmoebius ;)